projects/extension/sql/idempotent/015-vectorizer-api.sql

```diff
@@ -17,8 +17,10 @@ set search_path to pg_catalog, pg_temp
 create or replace function ai.create_vectorizer
 ( source pg_catalog.regclass
 , destination pg_catalog.name default null
+, loading pg_catalog.jsonb default null
```

Comment:
Again, should we keep the old create_vectorizer function and change its implementation to throw an error pointing out the changed API?

Reply:
thanks, as it has been decided to move forward instead, marking this as resolved.